cmd/cli/app/auth/offline_get.go

```diff
@@ -0,0 +1,85 @@
+//
+// Copyright 2024 Stacklok, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auth
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/stacklok/minder/internal/config"
+	clientconfig "github.com/stacklok/minder/internal/config/client"
+	"github.com/stacklok/minder/internal/util/cli"
+)
+
+// offlineTokenGetCmd represents the offline-token get command
+var offlineTokenGetCmd = &cobra.Command{
+	Use:   "get",
+	Short: "Retrieve an offline token",
+	Long: `The minder auth offline-token get command project lets you retrieve an offline token
+for the minder control plane.
+
+Offline tokens are used to authenticate to the minder control plane without
```

Comment:
I'm wondering if this should say "to the minder CLI", since the tokens are scoped to the CLI client. In theory tokens generated by different clients can have different characteristics, although in our current implementation they don't.